driver/test/java/io/crate/client/jdbc/integrationtests/BaseIntegrationTest.java

```diff
@@ -42,13 +42,12 @@
 @ThreadLeakScope(ThreadLeakScope.Scope.SUITE)
 public abstract class BaseIntegrationTest extends RandomizedTest {
 
-    private static final String[] CRATE_VERSIONS = new String[]{
+    private static final String[] CRATE_VERSIONS = new String[] {
+            "1.2.0",
```

Comment:
Since we only test for latest 4.x and latest 5.x, maybe we should consider removing the rest here?

Reply:
Yeah. I think it was good to see the test suite still succeeds on the older versions, but 9656e6c7cb8 now removes them, as suggested.